auto_tuning/proxy/src/proxy_common.hpp

```diff
@@ -0,0 +1,117 @@
+#ifndef SEISSOL_PROXY_PROXY_COMMON_DATATYPES_HPP
+#define SEISSOL_PROXY_PROXY_COMMON_DATATYPES_HPP
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include <stdexcept>
+
+enum Kernel {
+  all = 0,
+  local,
+  neigh,
+  ader,
+  localwoader,
+  neigh_dr,
+  godunov_dr
+};
+
+struct ProxyConfig {
+  unsigned cells{static_cast<unsigned>(1e5)};
```

Comment:
You can also use `1e5U instead of the static cast`

Reply:
It doesn't work for me. 1e5 if floating point value. Prefix `U` is meant to be for integer types. Could you share a code snippet if it works for you with gcc@10?

Reply:
Ah, right. My mistake. I'd use `100'000U;`, but your option is also good then.